tensorflow_addons/layers/max_unpooling_2d.py

```diff
@@ -59,6 +59,7 @@ def _max_unpooling_2d(updates, mask, pool_size=(2, 2), strides=(2, 2), padding="
     def func(updates, mask):
         mask = tf.cast(mask, "int32")
         input_shape = tf.shape(updates, out_type="int32")
```

Comment:
Do we need this line after adding line #62?

Reply:
It's still needed to ensure the function work with tensors of partially unknown shapes.

Reply:
acked.